DiscordBot.Commands/VoiceControlCommands/YoutubeCommand.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Discord.WebSocket;
+using DiscordBot.Commands.Core;
+using DiscordBot.Commands.Core.CommandAttributes;
+using DiscordBot.Commands.Helpers;
+using DiscordBot.Core.Voice;
+using Microsoft.Extensions.Logging;
+
+namespace DiscordBot.Commands.VoiceControlCommands
+{
+    [Name("yt")]
+    [Description("Plays audio from youtube")]
+    [Option("text", "Link to YouTube video or query (text) to search for video", CommandOptionType.String, true)]
```

Comment:
I feel like `video` would be a better name for this param, but that's a minor thing